File: problem-set-1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    //Do-while loop for user's input.
    float change;
    do
    {
        change = get_float("Change owed: ");
    }
    while (change <= 0);

    //Rounding change & naming it as cents.
    int cents = round(change * 100);

    //Simple division to get the number of coins used and the remainder of cents.
    int q1 = round(cents / 25);
    int q2 = round(cents % 25);

    int d1 = round(q2 / 10);
    int d2 = round(q2 % 10);

    int n1 = round(d2 / 5);
    int n2 = round(d2 % 5);

    int p1 = round(n2 / 1);

    //Addition of coins used & printing it out.
    int coins = q1 + d1 + n1 + p1;
    printf("Coins: %i\n", coins);
}

File: problem-set-6/cash/cash.py
import cs50

# Improvised do-while loop for user's input
co = cs50.get_float("Change owed: ")

while co <= 0:
    co = cs50.get_float("Change owed: ")


# Rounding change owed & storing it into co2
co2 = int(co * 100)


# Simple division to get the number of specific coins usable and the remainder of change left
q1 = int(co2 / 25)
q2 = co2 % 25

d1 = int(q2 / 10)
d2 = q2 % 10

n1 = int(d2 / 5)
n2 = d2 % 5

p1 = int(n2 / 1)


# Adding coins used & printing it out
change = q1 + d1 + n1 + p1
print(change)
